neutromeratio/parameter_gradients.py

```diff
@@ -201,7 +204,7 @@ def torchify(x):
             return torch.tensor(x, dtype=torch.double, requires_grad=True)
 
         states_with_samples = torch.tensor(self.mbar.N_k > 0)
-        N_k = torch.tensor(self.mbar.N_k, dtype=torch.double)
+        N_k = torch.tensor(self.mbar.N_k, dtype=torch.double, requires_grad=True)
```

Comment:
huh! wouldn't have expected that's required for the discrete "number of samples per state" array. this is added out of precaution, or because pytorch dislikes if it's missing?

Reply:
no, just remnant from my first debug session